src/commons.ts

```diff
@@ -2,10 +2,11 @@
 import * as vscode from "vscode";
 import { Environment } from "./environmentPath";
 import localize from "./localize";
-import { AutoUploadService } from "./service/autoUploadService";
+import { AutoUploadService } from "./service/autoUpload.service";

```

Comment:
May i know why you used `autoUpload.service` instead `autoUploadService` ( i mean changed file name).

Is it a standard somewhere? Then we can change all the services. Lets discuss before changing it.

Reply:
It's just a naming convention that I adopted while writing Angular apps (which also use TypeScript). If you don't like it though, I can revert it back to the previous name.

Reply:
> Angular apps (which also use TypeScript)

Yes i also used services in that way in Angular. Yes that's fine. Can you rename other services as well like above.

Reply:
Sure